packages/liquidator/index.js

```diff
@@ -54,6 +54,7 @@ async function run({
 }) {
   try {
     const { toBN } = web3.utils;
+    const getTime = () => Math.round(new Date().getTime() / 1000);
```

Comment:
Do the timestamps on Ethereum shave off three decimal points of precision compared to `Date()`?

Reply:
nope. get time returns time in milliseconds so this turns it into seconds :)